scripts/resources/idea/libraries/hp_dep.xml

```diff
@@ -21,14 +21,12 @@ under the License.
 <component name="libraryTable">
```

Comment:
I started making changes to this file, but realised that this is probably long out of date or dead, so avoided additional changes